Dockerfile.canary

```diff
@@ -12,4 +13,4 @@ RUN npm run playwright:install
 
 ENV TIMING_LOGS=true
 
-CMD ["npm", "run", "playwright:test:canary"]
+CMD ["./docker-canary.sh"]
```

Comment:
Doing as script instead of in-script (like CloudWatch) because the trace.zip isn't generated until after the test completes.